packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinSetupStrategies.tsx

```diff
@@ -1,94 +1,148 @@
+    }
+`;
 
 interface CoinjoinSetupStrategiesProps {
     account: Account;
 }
 
 export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProps) => {
-    const [strategy, setStrategy] = useState<keyof typeof COINJOIN_STRATEGIES>('default');
+    const [strategy, setStrategy] = useState<CoinJoinStrategy>('recommended');
+    const [connectedConfirmed, setConnectedConfirmed] = useState(false);
+    const [termsConfirmed, setTermsConfirmed] = useState(false);
+
     const actions = useActions({
         startCoinjoinSession,
     });
 
-    const selectedStrategy = COINJOIN_STRATEGIES[strategy];
+    const isCustom = strategy === 'custom';
+    const allChecked = connectedConfirmed && termsConfirmed;
+    const fee = (Number(account.balance) * 0.003).toString();
+
+    const reset = () => setStrategy('recommended');
+    const toggleConnectConfirmation = () => setConnectedConfirmed(!connectedConfirmed);
+    const toggleTermsConfirmation = () => setTermsConfirmed(!termsConfirmed);
+    const anonymize = () =>
+        actions.startCoinjoinSession(
+            account,
+            COINJOIN_STRATEGIES[strategy === 'custom' ? 'recommended' : strategy], // TODO: enable custom strategy
+        );
+
+    const tooltip = allChecked ? null : <Translation id="TR_ANONYMISATION_BUTTON_DISABLED" />;
 
     return (
-        <Wrapper>
-            <Content>
-                <Section style={{ flex: 1 }}>
-                    <SelectButton
-                        isActive={strategy === 'default'}
-                        variant="tertiary"
-                        onClick={() => setStrategy('default')}
-                    >
-                        Default
-                        {strategy === 'default' && <Icon icon="CHECK_ACTIVE" />}
-                    </SelectButton>
-                    <SelectButton
-                        isActive={strategy === 'speed'}
-                        variant="tertiary"
-                        onClick={() => setStrategy('speed')}
-                    >
-                        Speed
-                        {strategy === 'speed' && <Icon icon="CHECK_ACTIVE" />}
-                    </SelectButton>
-                </Section>
-                <Section>
-                    <CoinjoinSessionDetail account={account} {...selectedStrategy} />
-                </Section>
-            </Content>
-            <Button
-                onClick={() => actions.startCoinjoinSession(account, selectedStrategy)}
-                icon="ARROW_RIGHT_LONG"
-                alignIcon="right"
-            >
-                Start coinjoin
-            </Button>
-        </Wrapper>
+        <>
+            <StyledCard>
+                <Heading>
+                    <Translation id="TR_START_ANONYMISATION" />
+                </Heading>
+                <Row>
+                    <StyledCryptoAmountWithHeader
+                        header={
+                            <AmountHeading>
```

Comment:
Not sure on this one. Doing something like `div:first-child` - despite looking better - creates tightly coupled code that can break on change to the `CryptoAmountWithHeader`. This can be solved by exporting the Header element but that seems like an overkill.